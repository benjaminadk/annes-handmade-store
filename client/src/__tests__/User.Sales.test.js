import React from 'react'
import UserSales from '../components/User.Sales'
import { shallow } from 'enzyme'

test('User Sales Component', () => {
  const wrapper = shallow(<UserSales />)
  expect(wrapper).toMatchSnapshot()
})
