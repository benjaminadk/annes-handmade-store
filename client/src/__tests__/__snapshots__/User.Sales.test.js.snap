// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Sales Component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(UserSales) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "root": "UserSales-root-1",
        "sales": "UserSales-sales-3",
        "tableContainer": "UserSales-tableContainer-4",
        "title": "UserSales-title-2",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "root": "UserSales-root-1",
          "sales": "UserSales-sales-3",
          "tableContainer": "UserSales-tableContainer-4",
          "title": "UserSales-title-2",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
